src/routes/products/new.svelte

```diff
@@ -115,75 +115,69 @@
     persistToStorage();
   }
 
-  function createProduct() {
+  async function createProduct() {
     prefetch('/store');
     if (!product.name) {
       errors.name = true;
       errorMessage = 'Some fields are not filled out or filled out incorrectly.';
+      return;
```

Comment:
Perhaps it would be better to omit these returns to have all validation errors visible at the same time